libnavigation-core/src/main/java/com/mapbox/navigation/core/stops/ArrivalObserver.kt

```diff
@@ -15,11 +15,11 @@ interface ArrivalObserver {
      * Once [MapboxNavigation.navigateNextRouteLeg] has been called and returns true,
      * this observer will be notified of a stop arrival.
      */
-    fun onStopArrival(routeLegProgress: RouteLegProgress)
+    fun onWaypointArrival(routeLegProgress: RouteLegProgress)
 
     /**
      * Once the [RouteProgress.currentState] has reached [RouteProgressState.ROUTE_ARRIVED]
      * for the last stop, this will be called once.
```

Comment:
```suggestion
     * This will be called once the [RouteProgress.currentState] has reached [RouteProgressState.ROUTE_ARRIVED].
     * This means the device has reached the final destination on the route
```